docs/content/v2024.2/yugabyte-platform/back-up-restore-universes/disaster-recovery/_index.md

```diff
@@ -76,6 +76,42 @@ Blog: [Using YugabyteDB xCluster DR for PostgreSQL Disaster Recovery in Azure](h
 
 For more information on the YugabyteDB xCluster implementation and its limitations, refer to [xCluster implementation limitations](../../../architecture/docdb-replication/async-replication/#limitations).
 
+## Schema change modes
+
+You can set up xCluster DR to use one of two schema change modes, which differ in the steps required to replicate DDL operations from the DR primary to the DR replica.
+
+### Manual schema changes
+
+In this mode, table and index-level schema changes must be performed in a specific order as follows:
+
+1. The DR primary universe.
+1. The DR replica universe.
+1. In some cases, they must also be updated on the YugabyteDB Anywhere DR configuration.
+
+The exact sequence of these operations for each type of Schema Change (DDL) is described in [Manage tables and indexes](./disaster-recovery-tables/).
+
+### Semi-automatic schema changes
+
+Semi-automatic schema changes are {{<tags/feature/ea>}}. In this mode, table and index-level schema changes operations must be performed in the same order as follows:
+
+1. The DR primary universe.
+2. The DR replica universe.
```

Comment:
Should we have the constraint here that  ddl changes need to be performed one at a time? (Even though db does not have this constraint though @hari90  recommends this)

Reply:
It is covered in the tables section. This part is just an intro to how the two modes are different.